XamarinCommunityToolkitSample/Models/WelcomeSectionId.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace XamarinCommunityToolkitSample.Models
+{
+    public enum WelcomeSectionId
+    {
+        Behaviors,
+        Converters,
+        Extensions,
+        Views,
+        TestCases
+    }
+
+    public static class WelcomeSectionIdExtensions
+    {
```

Comment:
I would rename the file to include Extension

Reply:
Do you mean extract this class to the separate file?

Reply:
Yeah indeed

Reply:
Are you Ok with that as well?

That's like extension in swift :) or partial class in C#